app/konnect/configure/runtime-manager/runtime-groups/index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Runtime Groups
+no_version: true
+---
+
+Konnect manages runtime configuration in runtime groups. Each runtime group acts
+as a separate control plane and can manage runtime configurations independently
+of any other group. You can find a list of all runtime groups in your organization
+on the [Runtime Manager overview](https://konnect.konghq.com/runtime-manager/).
+
+* A **runtime group** is a collection of API connectivity runtime instances
+sharing the same configuration and behavior space.
+
+* A **runtime instance** is a single data plane node with a single instance of
+a runtime, such as Kong Gateway. Runtime instances service traffic for the runtime
+group. All runtime instances in one runtime group
+must be of the same type. Currently, only Kong Gateway runtime types are supported.
+
+Every organization has one default runtime group. With an Enterprise subscription,
+you can configure two additional custom runtime groups in the same Konnect
+account to manage runtime instances and their configuration in any groupings
+you want.
+
+_<diagram - do we have a diagram of how runtime groups work somewhere?>_
+
+## Default runtime group
+
+The default runtime group is the foundational group in Konnect. Every
+organization starts with one default group.
+
+This group can be renamed, but it can't be deleted, and its status as the default
+group can't be changed.
```

Comment:
Is there going to be some identifier for the default group besides the name?
If the name can be changed, then how would a future admin know which group is the default?

Reply:
The "default" RG cannot be renamed.

At the moment, the plan is to use the name as the identifier in Helium. It is a known limitation and slightly inconvenient to the user. The hope is to remove default group in the long run, but will be few months away.

Reply:
Sorry, wiki was out of date. (I've added a comment in the wiki)